audio/kwave/Makefile

```diff
@@ -16,15 +16,14 @@ LIB_DEPENDS=	libaudiofile.so:audio/libaudiofile \
 		libsamplerate.so:audio/libsamplerate \
 		libvorbisenc.so:audio/libvorbis
 
-USES=		cmake compiler:c++11-lang desktop-file-utils gettext kde:5 \
-		localbase:ldflags magick:6,build pkgconfig qt:5 tar:xz xorg
+USES=		cmake compiler:c++11-lang desktop-file-utils gettext kde:6 \
+		localbase:ldflags magick:6,build pkgconfig qt:6 tar:xz xorg
```

Comment:
I don't think this has needed magick:6 for quite a while. It previously used `convert` to build the docs.